### PR TITLE
Fixed UK data download

### PR DESCRIPTION
**Pull request by seabbs**
* Upstream whitespace has started to creep into the data source meaning that cases was detected as a character string.
      * Stripped out whitespace using `stringr`
      * Converted to numeric
      * Dropped missing numeric entries
* Also seeing missing dates and I am dropping. I may have missed a change in format here so could be dropping data accidentally. 

Checking unique dates (below) this looks like a real NA so safe to remove. 

```
 unique(cases$Date)
 [1] "07/03/2020" "08/03/2020" "09/03/2020" "10/03/2020" "11/03/2020" "12/03/2020" "13/03/2020" "14/03/2020"
 [9] "15/03/2020" "16/03/2020" "17/03/2020" "18/03/2020" "19/03/2020" "20/03/2020" "21/03/2020" "22/03/2020"
[17] "23/03/2020" "24/03/2020" "25/03/2020" "26/03/2020" "27/03/2020" "28/03/2020" "29/03/2020" "30/03/2020"
[25] "31/03/2020" "01/04/2020" "02/04/2020" NA   
```

**This is needed for gov forecasts tonight so could someone check asap and carefully :) as I may have goofed.**

### CODE REVIEW

**Comment by hamishgibbs** [comment]
Checking it out now

**Comment by hamishgibbs** [comment]
I have looked at the parsed dates and the data from the source and it looks legit to drop missing date
and case values to me. Want another pair of eyes on it before merging?

**Review by hamishgibbs — APPROVED** [review]

**Review by jhellewell14 — APPROVED** [review]

**Comment by seabbs** [comment]
Thanks for reviewing!

**Pull request merged**
